scripts/install_moodle.sh

```diff
@@ -307,8 +317,17 @@ echo_info "Done."
 echo_title "Download and extract Moodle files."
 ###############################################################################
 # Ref.: https://download.moodle.org/releases/supported/
-echo_action "Downloading Moodle 3.8.4 tar file..."
-wget https://download.moodle.org/download.php/direct/stable38/moodle-3.8.4.tgz
+echo_action "Downloading Moodle 3.10.1 tar file..."
+moodle_zip_file_url=https://download.moodle.org/download.php/stable310/moodle-3.10.1.tgz
+moodle_zip_file=$(basename ${moodle_zip_file_url})
+wget ${moodle_zip_file_url} -O ${moodle_zip_file}
+echo_info "Done."
+
+echo_action "Verifying downloaded file integrity..."
+echo "SHA256(${moodle_zip_file})= 547973f2dc2ca7b992c193dab4991c88768338fd2bd04b9d786e62de08cf513f" | sha256sum --check
```

Comment:
hard-coding the result?

Reply:
Yes. We are not computing sha256 value, we are checking it.